### PR TITLE
Add input validation and UDP testing

### PR DESCRIPTION
**Pull request by SalDaniele**
This commit introduces some validation of user input and sets reasonable defaults on option arguments.

It also adds the option for iperf UDP testing.

### CODE REVIEW

File: iperf.py [hunk 1]
```diff
@@ -122,14 +126,36 @@ def stop(self):
...
         if r.returncode != 0:
             logger.info(r)
-
         data = json.loads(r.out)
+        if self.test_type == TestType.IPERF_TCP:
+            self.print_tcp_results(data)
+        if self.test_type == TestType.IPERF_UDP:
+            self.print_udp_results(data)
+    
+    def print_tcp_results(self, data: dict):
```
Comment (wizhaoredhat) [inline iperf.py:135]: I guess the TCP results needs work as well to display as much info as UDP results.

File: testSettings.py [hunk 2]
```diff
@@ -59,7 +60,7 @@ def _is_same_node_test(self, test_id: int) -> bool:
 
     def _server_test_to_persistent(self, test_id: int) -> bool:
         #TODO: add logic to determine when this is required
-        return False
+        return True
```
Comment (wizhaoredhat) [inline testSettings.py:63]: Let's try to run the tests without persistent at first. I think it is the harder option.

File: common.py [hunk 3]
```diff
@@ -2,6 +2,14 @@
 from enum import Enum
 
 
+FT_BASE_IMG = "quay.io/wizhao/ft-base-image:0.9"
+TFT_BASE_IMG = "quay.io/wizhao/tft-tools:0.3"
```
[the remark this answers is not in the document]
Comment (SalDaniele) [reply common.py:6]: Updated

File: iperf.py (hunk 1 above)
Comment (SalDaniele) [reply iperf.py:135]: Yes, I can add this in a future PR

**Pull request merged**